src/Umbraco.Web.UI.Client/src/views/common/infiniteeditors/mediapicker/mediapicker.controller.js

```diff
@@ -98,21 +98,30 @@ angular.module("umbraco")
                         gotoStartNode();
                     }
                 } else {
-                    //if a target is specified, go look it up - generally this target will just contain ids not the actual full
-                    //media object so we need to look it up
+                    // if a target is specified, go look it up - generally this target will just contain ids not the actual full
+                    // media object so we need to look it up
                     var id = $scope.target.udi ? $scope.target.udi : $scope.target.id;
-                    var altText = $scope.target.altText;
-                    entityResource.getById(id, "Media")
+
+                    // ID of a UDI or legacy int ID still could be null/undefinied here
+                    // As user may dragged in an image that has not been saved to media section yet
+                    if(id){
+                        entityResource.getById(id, "Media")
                         .then(function (node) {
-                            $scope.target = node;
+                            //$scope.target = node; //Not 100% sure why we override it here?!
                             if (ensureWithinStartNode(node)) {
                                 selectImage(node);
-                                $scope.target.url = mediaHelper.resolveFile(node);
-                                $scope.target.altText = altText;
+                                //$scope.target.url = mediaHelper.resolveFile(node);
```

Comment:
Same for this - why was we calling this?
Also this returns nothing for me & thus the textbox for the URL is empty & was getting no preview & if we were to edit/update the ALT tag in this state. It would previously set the image source to `nothing.jpg`